.github/workflows/pr-verifier.yaml

```diff
@@ -0,0 +1,16 @@
+name: PR Verifier
+
+on:
+  pull_request_target:
+    types: [opened, edited, reopened, synchronize]
+
```

Comment:
Now we don't have any permissions, which is also bad. If it would be my guess, we need 
```yaml
permissions:
  contents: read
```
since we're reading the PR title.